improver/calibration/rainforest_calibration.py

```diff
@@ -355,7 +356,8 @@ def _prepare_features_dataframe(self, feature_cubes: CubeList) -> DataFrame:
                 Cubelist containing the independent feature variables for prediction.
 
         Returns:
-            Dataframe containing flattened feature variables.
+            array containing flattened feature variables,
```

Comment:
I think the preferred formatting on multiple return items is:
```
"""
    Returns:
        - First item.
        - Second item.
"""
With `-` to denote each item. I'm not exactly sure how fussy the docs are, but better to follow the style guide closely.

Reply:
I'm now returning only 1 item.